### DISCUSSION TITLE
Inquiry about an accessibility discussion.

### DISCUSSION THREAD

**Discussion by Ohadha-bobly** [w3c/wcag#3857] [Q&A]
What should I talk about when I'm asked about "Fostering a web landscape that caters to all" in the context of accessibility?

**Comment by detlevhfischer** [comment]
Your question is so general that it is hard to give an answer beyond pointing you to introductory resources like: 
https://www.w3.org/WAI/resources/#accessibility-fundamentals

That should give you enough of a starting point to put together some basic information.
